Running website in docker doesn't work anymore

`docker-compose up --build` doesn't work.

it probably has something to do with how docker mounts volumes. the problem started appearing after we moved `client` into `client/visualizer`

error message:
```
frontend_1  | env: can't execute 'node': Exec format error
frontend_1  | npm ERR! code ELIFECYCLE
frontend_1  | npm ERR! errno 126
frontend_1  | npm ERR! bc20-client@1.1.7 prod: `webpack`
frontend_1  | npm ERR! Exit status 126
frontend_1  | npm ERR! 
frontend_1  | npm ERR! Failed at the bc20-client@1.1.7 prod script.
frontend_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
frontend_1  | 
frontend_1  | npm ERR! A complete log of this run can be found in:
frontend_1  | npm ERR!     /root/.npm/_logs/2019-11-20T00_29_38_226Z-debug.log
frontend_1  | npm ERR! code ELIFECYCLE
frontend_1  | npm ERR! errno 126
frontend_1  | npm ERR! battlecode-app@0.1.0 startx: `cd ../client/visualizer && npm run prod && cp -r bc20 ../../frontend/public && cd ../../frontend && BROWSER=none react-scripts start`
frontend_1  | npm ERR! Exit status 126
frontend_1  | npm ERR! 
frontend_1  | npm ERR! Failed at the battlecode-app@0.1.0 startx script.
frontend_1  | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -9,6 +9,9 @@ services:
     volumes:
       - ./frontend:/frontend
       - ./client:/client
+      #issue with which node_modules folder docker sees (see https://github.com/barrysteyn/node-scrypt/issues/148)
+      - ./client/visualizer/node_modules
+      - ./client/playback/node_modules
       - ./schema:/schema
     ports:
       - 3000:3000

